ui/app/pages/confirm-add-token/confirm-add-token.component.js

```diff
@@ -103,10 +104,27 @@ export default class ConfirmAddToken extends Component {
               className="page-container__footer-button"
               onClick={() => {
                 addTokens(pendingTokens).then(() => {
+                  const pendingTokenValues = Object.values(pendingTokens)
+                  console.log(pendingTokenValues)
+                  pendingTokenValues?.forEach((pendingToken) => {
+                    this.context.trackEvent({
+                      event: 'Token Added',
+                      category: 'Wallet',
+                      excludeMetaMetricsId: true,
+                      properties: {
+                        token_symbol: pendingToken.symbol,
+                        token_contract_address: pendingToken.address,
+                        token_decimal_precision: pendingToken.decimals,
+                        custom: pendingToken.isCustom,
```

Comment:
I'm not sure whether `isCustom` is ever set for suggested tokens :thinking:  Maybe we should instead be communicating that this was a suggested token.

Reply:
We discussed this in the tmp-metamask-extension-analytics channel, and decided to track this as two separate properties: `unlisted` and `source`